CONTRIBUTING/RELEASING.md

```diff
@@ -0,0 +1,534 @@
+
+Done! 🚀
+
+## Releasing Locally in an Emergency 🚨
+
+Things can fail, code can break, and bugs can exist. When automation is broken, there may be a need for an emergency escape hatch to release new fixes. In such a situation, it's valid to run the whole release process locally instead of relying on pull requests and workflows. You don't need to create pull requests or split preparation and publishing; you can do it all at once, but make sure you still follow the correct branching strategy.
+
+You need a token to the npm registry to publish (set as `YARN_NPM_AUTH_TOKEN`). Currently, @shilman and @ndelangen hold the keys to that castle.
+
+You can inspect the workflows to see what they are running and copy that, but here is a general sequence of steps to mimic the automated workflow. Feel free to deviate from this as needed.
+
+1. Create a new branch from either `next` (prereleases) or `main` (patches)
+2. Get all tags: `git fetch --tags origin`
+3. `cd scripts`
+4. (If patch release) Cherry pick:
+   1. `yarn release:pick-patches`
+   2. Manually cherry pick any necessary patches based on the previous output
+5. Bump versions: `yarn release:version --verbose --release-type <RELEASE_TYPE> --pre-id <PRE_ID>`
+6. To see a list of changes (for your own to-do list), run `yarn release:generate-pr-description --current-version <CURRENT_VERSION> --next-version <NEXT_VERSION_FROM_PREVIOUS_STEP> --verbose`
+7. Write changelogs: `yarn release:write-changelog <NEXT_VERSION_FROM_PREVIOUS_STEP> --verbose`
+8. `git add .`.
+9. Commit changes: `git commit -m "Bump version from <CURRENT_VERSION> to <NEXT_VERSION_FROM_PREVIOUS_STEP> MANUALLY"`
+10. Merge changes to the release branch:
+    1. `git checkout <"latest-release" | "next-release">`
+    2. `git merge <PREVIOUS_BRANCH>`
+    3. `git push origin`
+11. (If automatic publishing is still working, it should kick in now and the rest of the steps can be skipped)
+12. `cd ..`
+13. Install dependencies: `yarn task --task=install --start-from=install`
+14. Publish to the registry: `YARN_NPM_AUTH_TOKEN=<NPM_TOKEN> yarn release:publish --tag <"next" OR "latest"> --verbose`
+15. (If patch release) `yarn release:label-patches`
+16. Manually create a GitHub Release with a tag that is the new version and the target being `latest-release` or `next-release`.
+17. Merge to core branch:
+    1. `git checkout <"next"|"main">`
+    2. `git merge <"next-release"|"latest-release">`
+    3. `git push origin`
+18. (If patch release) Sync `CHANGELOG.md` to `next` with:
+    1. `git checkout next`
+    2. `git pull`
+    3. `git checkout origin/main ./CHANGELOG.md`
+    4. `git add ./CHANGELOG.md`
+    5. `git commit -m "Update CHANGELOG.md for v<NEXT_VERSION>"`
+    6. `git push origin`
+
+## Canary Releases
+
+Not implemented yet. Still work in progress, stay tuned.
+
+## Versioning Scenarios
```

Comment:
I like the breakdown of different types of releases here. But the cadence of each is something we should talk about and agree to before committing to it.

Reply:
That's fine by me. I didn't intend for this to be a commitment but more of contextual knowledge to the reader. I've removed it.